SystemServices/CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.0.2] - 2022-08-24
+### Fixed
+- RDKTV-19132 Update SystemServices to call container_setup.sh script after deleting persistent data
```

Comment:
Pls remove ticket details (RDKTV-19132). We don't want to add JIRA ticket details in changelog. Also pls update the patch version in the macro so that it gets reflected in the status API - https://github.com/rdkcentral/rdkservices/blob/main/SystemServices/SystemServices.cpp#L69

Reply:
Done